### PR TITLE
Add 'what has changed in 5.11.0'

### PR DESCRIPTION
**Pull request by sherenewso2**
## Purpose
Resolves:
https://github.com/wso2/product-is/issues/8343
https://github.com/wso2/product-is/issues/8446
https://github.com/wso2/product-is/issues/9058
https://github.com/wso2/product-is/issues/10136
https://github.com/wso2/product-is/issues/10090

### CODE REVIEW

File: en/docs/setup/migrating-what-has-changed.md [hunk 1]
```diff
@@ -1,387 +1,260 @@
...
+
+Complete the following steps for the CORS migration.
 
-## Handling Internal Scopes
-From IS 5.10.0 onwards, internal scopes validation while accessing internal resources will be enabled by default, 
-but this will break existing applications using some APIs such as scim2/Me.
+1. Remove any CORS configurations defined in the `<IS_HOME>/repository/resources/conf/tomcat/web.xml` file. 
+To do this, remove the whole tag under the filter class named `com.thetransactioncompany.cors.CORSFilter`. 
+
+2. In order to complete the CORS migration, any CORS configurations defined at the `web.xml` file should be reconfigured in the `deployment.toml` file. The following table shows how the old configurations in the `web.xml.j2` file are mapped to the new ones in the `deployment.toml` file.
```
Comment (Buddhimah) [inline en/docs/setup/migrating-what-has-changed.md:115]: change web.xml.j2 to web.xml

**Review by Buddhimah — APPROVED** [review]

File: en/docs/setup/migrating-what-has-changed.md [hunk 2]
```diff
@@ -1,387 +1,260 @@
...
-    below configuration in the `deployment.toml` file located at `<IS_HOME>/repository/conf/` directory.
-    ```toml
-    [authentication.authenticator.sms_otp.parameters]
-    UseInternalErrorCodes = false
-    ```
-## New User Store Managers with built in unique ID support. 
+The token binding validation is a feature that was introduced with WSO2 IS 5.10.0. Intially, if `tokenBindingType` is provided when creating the OpenID Connect service provider, the token binding validation is enabled by default. With the WSO2 IS WUM-updated pack, you can disable it by unticking the **Validate token bindings** property. However, if this property is not available for the existing service providers, token binding will still be enabled by default. 
+
+From 5.11.0 onwards, this behaviour has been changed. The property is available for all new service providers. For the exisiting service providers that do not have this property, token binding validation is disabled. To maintain backward compatibility, the property will be enabled during migration for those exisiting service providers.
```
Comment (Buddhimah) [inline en/docs/setup/migrating-what-has-changed.md:177]:
```suggestion
From 5.11.0 onwards, this behavior has been changed. The property is available for all new service providers. For the existing service providers that do not have this property, token binding validation is disabled. 
To maintain backward compatibility, the property will be enabled during migration for those existing service providers which has the `tokenBindingType` as **cookie**
```

File: en/docs/setup/migrating-what-has-changed.md [hunk 3]
```diff
@@ -1,387 +1,277 @@
...
-with the addition of new properties to the service provider model object. Therefore SOAP API consumers should 
-regenerate the stub or upgrade the dependency version of the stub.
+## Disabled appending params to OAuth2/OIDC error responses
+
+Appending additional parameters to the OAuth2/OIDC error response has been disabled by default. As a result, the parameters 'sp' and 'tenant_domain' which was set to the callback url in error responses in previous WSO2 IS versions will also not be available. If any configured error param needs to be sent back to the callback url in an error response, add the following config to the `deployment.toml` file.
+
+```toml
+[oauth.allow_additional_params_from_error_url]
+enable = true
+```
```
Comment (dewniMW) [inline en/docs/setup/migrating-what-has-changed.md:261]: @sherenewso2 this config should be changed as:

```
[oauth]
allow_additional_params_from_error_url = false
```

Comment (sherenewso2) [reply en/docs/setup/migrating-what-has-changed.md:261]: fixed with https://github.com/wso2/docs-is/pull/2168

**Pull request merged**